framework/sandstone_tests.h

```diff
@@ -28,14 +28,16 @@ extern struct test __stop_tests;
 extern struct test mce_test;
 
 struct test_set_cfg {
-    bool ignore_unknown_tests;  /* whether an error should be reported if
-                                   there's no test matching the specified name,
-                                   either on command-line or in a test list. */
-    bool randomize;             /* iterate through the tests in random order. */
-    bool cycle_through;         /* start over when an iteration through the test
-                                   set has finished. */
-    bool is_selftest;           /* whether to use regular or selftests as the
-                                   source of the tests. */
+    bool ignore_unknown_tests;          /* whether an error should be reported if
+                                           there's no test matching the specified name,
+                                           either on command-line or in a test list. */
+    bool randomize;                     /* iterate through the tests in random order. */
+    bool cycle_through;                 /* start over when an iteration through the test
+                                           set has finished. */
+    bool is_selftest;                   /* whether to use regular or selftests as the
+                                           source of the tests. */
+    uint64_t enabled_features_mask;     /* features requested by the user. */
+    uint64_t disabled_features_mask;    /* features requested by the user. */
```

Comment:
Ditto on `decltype`

Reply:
Done by changes in `x86simid_generate.pl`